infiniband.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- hosts:
+  - cluster_login
+  - cluster_control
+  - cluster_batch
+  tasks:
+    - name: galexrt.kernel-modules
+      include_role: 
+        name: galexrt.kernel-modules
+      vars:
+        kernel_modules:
+          - name: ib_ipoib
+            blacklist: false
+          - name: mlx5_ib
+            blacklist: false
+        kernel_load_file: mlnx-ipoib.conf
```

Comment:
Are we missing this file? or does the package install it?

Reply:
I believe the `galexrt.kernel-modules` creates this: https://github.com/galexrt/ansible-kernel-modules/blob/83f14eed4519261f9debc8f254a30ab2d497d3b2/tasks/load-modules.yml#L14 (just overriding the default)